FallHandling.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using BattleTech;
+
+namespace CharlesB
+{
+    public class FallHandling
+    {
+        private static readonly string KnockdownPhrasePath = Path.Combine(Core.ModDirectory, "phrases.txt");
```

Comment:
I don't think this is gonna work because ModDirectory is gonna be null when this is assigned. You can make it a lambda `=>` (or a function), but not a variable.